HACKING.md

```diff
@@ -56,6 +56,11 @@ Creating VM images locally (not necessary for running tests) needs the following
 
     $ sudo dnf install virt-install
 
+Updating the `node_modules` (in case you need to modify `package.json`)
```

Comment:
please squash the "document t/n-m" commit into the previous one that introduces it.

Reply:
Agreed.